javascript/javascript2/week1/styles.css

```
@@ -269,3 +269,82 @@ ul li {
 .search-button:hover {
   background: #942f2f;
 }
+
+.countdown {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #ffffff;
+  padding: 30px;
+  border-radius: 12px;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+  max-width: 400px;
+  margin: 30px auto;
+  text-align: center;
+}
+
+#minutes {
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  font-size: 16px;
+  width: 150px;
+  margin-bottom: 15px;
+  transition: border-color 0.3s ease;
+}
+
+#minutes:hover,
+#minutes:focus {
+  border-color: #b33a3a;
+  box-shadow: 0 0 5px rgba(179, 58, 58, 0.2);
+  outline: none;
+}
+
+#start-timer {
+  background: #b33a3a;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
+#start-timer:hover {
+  background: #942f2f;
+}
+
+#timer-display {
+  font-size: 2em;
+  margin-top: 20px;
+  color: #333;
+  font-weight: bold;
+}
+
+.timer-overlay {
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  background: rgba(255, 255, 255, 0.95);
+  padding: 15px;
+  border-radius: 8px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.15);
+  z-index: 1000;
+}
+
+.time-spent {
+  text-align: center;
+}
```

No need to introduce an extra div, just to add the alignment property there.
```suggestion
  z-index: 1000;
  text-align: center;
}
```